src/clevertap.js

```diff
@@ -630,7 +643,7 @@ export default class CleverTap {
     this.notifications._processOldValues()
   }
 
-  debounce (func, delay) {
+  #debounce (func, delay) {
```

Comment:
since we are touching this piece of code, lets add a default delay here of 300 ms. We do not need to pass delay then